### ISSUE TITLE
ETHOS support for Older x10 ACCST IXJT Modules.

### ISSUE THREAD

**Issue by ETHOSWarrier** [FrSkyRC/ETHOS-Feedback-Community#2768]
Ok..Having spent over £1000 GBP on FrSky Transmitters(x20,x20s and x18SE) and Probably the same again  on ACCESS and ACCST receivers I think myself and similar others have earned the right to ETHOS support for our older tx.....I have a lovely X10 with and unused  IXJT internal module that I have HAD to buy  an external XJT frky module for (again spent good money on FrSky products) and use it as a backup radio and boat /car radio on the external XJT module.

I thinks its only fair now for ETHOS to be able to support me in my journey with my older radio.Having been "converted "to ETHOS I hate  using other OS on my x10 to allow the internal rf module to function.Its a backup radio and has all my models on from my x20 and x18 etc 

I know its possible in ETHOS as it supports the external module so a little mod  to ETHOS and I will be a  happy and fully fulfilled ETHOS man without having to resort to FROS ,OpenTx or EdgeTX  for my old x10!

So as the title asks...........Please,Please Please....ETHOS Warrior.

**Comment by andika207** [comment]
They haven't closed this issue but the bot will do it within 90 days. 
if they had any interest on supporting your obsolete Tx they would had done it from the beggining.
3 active radios and you need a 4th as backup ? just sell it and grab another X20 LOL

**Comment by Luznatural** [comment]
I fully agree with Ethos Warrior.

I have only two radios: the backup is my reliable X9D plus, and my daily is an original X12S with internal iXJT. I remember that Frsky announced the radio as "Lets you set the limits". It represented a huge investment only 4 years ago, is one of the best radios ever made by Frsky and has many many advantajes over X18s or X20S.

Ethos supports the very same radio with the ISRM and PARA modules, but does not support the original configuration (I remind you that this radio has internal GPS, BT, and a third analog gimbal that I personaly use for many things in my scale models (I have a plane where the pilot moves the head and even perform a military salute. I have another one that moves the dorsal gun turret, etc). This configuration is supported by FrskyOS but not by Ethos. You cannot easily do this with X18S and X20S!

I decided not to incorporate the ACCESS and PARA upgrades, as they make you loose the capability of using the older Frsky ACCST receivers (I have about 30 X6R and X8R). Actually, I performed the touch screen upgrade, but I am forced using Edgetx due to lack of support from Ethos, that I like more due to its simplicity over EdgeTx.

I simply do not understand this policy from Frsky. And the rest of Frsky traditional users agree with Ethos Warrior and me.

I personally think that Frsky is doing the very same error than Futaba, JR, and other companies did in the past and lost many clients in the process.
